docs/RouteRequest.md

```diff
@@ -192,22 +192,34 @@ and in the [transferRequests in build-config.json](BuildConfiguration.md#transfe
 **Since version:** `2.0` ∙ **Type:** `duration` ∙ **Cardinality:** `Optional` ∙ **Default value:** `"PT0S"`   
 **Path:** /routingDefaults 
 
-The minimum extra time after exiting a public transport vehicle.
+The time safety margin when alighting from a vehicle.
+
+This time slack is added to arrival time of the vehicle before any transfer or onward travel.
+
+The sole reason for this is to avoid missed connections when there are minor schedule variations. This
+parameter is intended to be set by agencies not individual users. In general it is better to use
+`boardSlack` - see its documentation for details. However, for specific modes, like airplane and
+subway, that need more time than others, this is also configurable per mode with `alightSlackForMode`.
 
-The slack is added to the time when going from the transit vehicle to the stop.
 
 <h3 id="rd_boardSlack">boardSlack</h3>
 
 **Since version:** `2.0` ∙ **Type:** `duration` ∙ **Cardinality:** `Optional` ∙ **Default value:** `"PT0S"`   
 **Path:** /routingDefaults 
 
-The boardSlack is the minimum extra time to board a public transport vehicle.
+The time safety margin when boarding a vehicle.
 
-The board time is added to the time when going from the stop (offboard) to onboard a transit
-vehicle.
+The board slack is added to the passenger's arrival time at a stop before boarding evaluating which
+vehicles can be boarded.
 
-This is the same as the `transferSlack`, except that this also applies to to the first
-transit leg in the trip. This is the default value used, if not overridden by the `boardSlackList`.
+The sole reason for this is to avoid missed connections when there are minor schedule variations. This
```

Comment:
Again, connection to me refers more to a transfer than potentially the only boarding but I'm not a native speaker.

Reply:
I agree, however please see above.